data/dashboard.yml

```diff
@@ -41,6 +41,17 @@
         - name: Repositories
           url: /repos.html
           description: All the applications and other repositories we use to build GOV.UK.
+    - name: GOV.UK documentation
+      sites:
+        - name: 2nd-line Support Manuals
+          url: /manual/2nd-line.html
+          description: The documentation for GOV.UK's "Second-line Support"
+        - name: Developer Documentation
+          url: /manual.html
+          description: The contents page of GOV.UK's Developer Documentation
```

Comment:
I found this confusing as the "Developer docs" encompass more than the just the "Manuals" sections and you can't get to everything from the Manuals contents page (e.g. Apps, Repos etc)

Reply:
Good point!